test/test_ops.py

```diff
@@ -309,7 +308,7 @@ def test_variant_consistency_jit(self, device, dtype, op):
                     check_against_reference(self,
                                             script_fn,
                                             func,
-                                            op.output_func,
+                                            lambda x: x,
```

Comment:
@mruberry Actually this PR drops one test by passing this lambda. The code in check_against_reference that compares the grad of script and reference func here
https://github.com/pytorch/pytorch/blob/9f93d829079dc9f6709e4514b354e36149634627/torch/testing/_internal/common_jit.py#L43
Will ignore the output of slogdet because we removed the itemgetter and now the output type is no longer a Tensor. Also, linalg.eigh had an ouput_func that still returned a tuple so that was also skipped in this jit grad test.